geomstats/geometry/spd_matrices.py

```diff
@@ -61,6 +67,88 @@ def random_tangent_vec_uniform(self, n_samples=1, base_point=None):
 
         return tangent_vec
 
+    def set_eigensummary(self, eigenspace, eigenvalues):
+        """
+        Set the current eigenspace for a specific family of matrices.
+
+        :param eigenspace: array-like, shape=[n, n]:
+        Current eigenspace.
+        :param eigenvalues: array-like, shape=[n, n]:
+        Current eigenvalues.
+        """
+        self.eigensummary = EigenSummary(
+            eigenspace=eigenspace, eigenvalues=eigenvalues)
+
+    def get_eigensummary(self):
+        """
+        Get the current eigenspace for a specific family of matrices.
+
+        :return Eigen namedtuple : Current eigenspace and eigenvalues.
+        """
+        return self.eigensummary
+
+    def random_gaussian(
+            self, mean_spd=None, var_rotations=None, n_samples=1):
+        """
+        Define a Gaussian random sample of SPD matrices.
+
+        (definition is debatable)
+        'mean_spd' is the mean SPD matrix; 'var_rotations' is the
+        scalar variance by which the mean is rotated.
+        """
+        n = self.n
+
+        # Should make it possible to simply perform EIG operation
```

Comment:
No left-over comment: You can create an issue on GitHub, that references this line, with this question.

Reply:
Replaced with a `NotImplementedError`.